app/javascript/components/widgets/weather/moon-phase.jsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { colors } from '../../../lib/theme';
+
+const iconRender = (posX, posY, radius, phase) => {
+  switch (true) {
```

Comment:
I think we want

```javascript
switch (phase) {
  case 'newMoon':
    ....
}
```

Reply:
I use includes() to reduce repetition for the Gibbous and Quarter shapes, how do you recommend I convert that to this style?

Reply:
if/else if/else conditions in that case. `switch` suggests an exact match for cases.

i suspect there is some refactoring opportunities here as well. Defining the propTypes for `phase` might help illustrate the options.